pymongo/aggregation.py

```diff
@@ -134,7 +134,7 @@ def get_cursor(
         self,
         session: ClientSession,
         server: Server,
-        sock_info: SocketInfo,
+        connection: Connection,
```

Comment:
What do you think about naming this "conn: Connection" to be more succinct everywhere?